### PR TITLE
Data zipper. Added option to specify site and provided more feedback in terminal

### LINKED ISSUE #116: Make data zipper function

### PR DESCRIPTION
**Pull request by hvjay**

### CODE REVIEW

**Comment by hvjay** [comment]
Example output file name: `~/Desktop\visual-N170_local_ntcs-29_07_2021_13:14_zipped`
~/Desktop/{experiment}_{site}-{day_month_year}_{hour:minute}_zipped

File: eegnb/cli/introprompt.py [hunk 1]
```diff
@@ -93,6 +93,14 @@ def exp_prompt() -> str:
 
     return exp_selection
 
+def site_prompt() -> str:
+    print("\nPlease select which specific folder you would like to zip. Default 'local_ntcs' :")
```
Comment (oreHGA) [inline eegnb/cli/introprompt.py]: I think it's may be a little hard for a regular user to know what possible options exist.

Maybe mention this identifies the "experiment 'subfolder'"

Comment (oreHGA) [reply eegnb/cli/introprompt.py]: On seconds thought... a possible option here could be passing the "experiment" value into site_prompt() to check for the existing "subfolders / valid 'site'" options so the user knows which one to pick using os.listdir()

**Review by oreHGA — APPROVED** [review]

File: eegnb/cli/introprompt.py (hunk 1 above)
Comment (oreHGA) [reply eegnb/cli/introprompt.py]: Awesome. Thanks for adding @HVjay

**Pull request merged**